newIDE/app/src/EventsFunctionsExtensionEditor/OptionsEditorDialog/ExtensionOptionsEditor.js

```diff
@@ -217,16 +218,10 @@ export const ExtensionOptionsEditor = ({
               forceUpdate();
             }}
           />
-          <TextField
-            floatingLabelText={
-              <Trans>Author (Name, email or GitHub handle)</Trans>
-            }
-            value={eventsFunctionsExtension.getAuthor()}
-            onChange={(e, text) => {
-              eventsFunctionsExtension.setAuthor(text);
-              forceUpdate();
-            }}
-            fullWidth
+          <UsersAutocomplete
+            userIds={eventsFunctionsExtension.getAuthorIds()}
+            floatingLabelText={t`Authors`}
```

Comment:
Excellent thing to pass this as props. Turns out that you can use `<Trans></Trans>`. 
We favor this over `t`  because:
- you don't need to import <I18n> to use <Trans> in the final component displaying the message. 
- this enables advanced stuff like: `<Trans>Hello {something}<Trans>` (that will be turned into a translation like `Hello <0></0>` by js-lingui).
- this enables to pass other components (we don't want this for labels, but for helper texts it could be useful).